Tests/TuistGeneratorTests/Utils/SettingsHelperTests.swift

```diff
@@ -200,9 +200,9 @@ final class SettingsHelpersTests: XCTestCase {
     }
 
     func testSettingsProviderPlatform() {
-        XCTAssertEqual(subject.settingsProviderPlatform(.test(platform: .iOS)), .iOS)
-        XCTAssertEqual(subject.settingsProviderPlatform(.test(platform: .macOS)), .macOS)
-        XCTAssertEqual(subject.settingsProviderPlatform(.test(platform: .tvOS)), .tvOS)
+        XCTAssertEqual(subject.settingsProviderPlatform(.iOS), .iOS)
+        XCTAssertEqual(subject.settingsProviderPlatform(.macOS), .macOS)
+        XCTAssertEqual(subject.settingsProviderPlatform(.tvOS), .tvOS)
     }
 
     func testSettingsProviderProduct() {
```

Comment:
What about adding a test for the new [`overlay`](https://github.com/tuist/tuist/pull/5369/files#diff-6a98541154e3e3ae25cc0193e83af3760e1dd95f8c579ceb4a7ab43d8a429c02R29) function?

Reply:
yup, i ended up keeping it so i'll add this to my todo list